<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventOptionsDialog</class>
 <widget class="QDialog" name="EventOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="timeGroupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Event Time Interval</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="endtimeDateTimeEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="endtimeLabel">
        <property name="text">
         <string>End Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="starttimeDateTimeEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="starttimeLabel">
        <property name="text">
         <string>Start Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="magDepthGroupBox">
     <property name="title">
      <string>Magnitude and Depth</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="2">
       <widget class="QLineEdit" name="maxdepthLineEdit">
        <property name="text">
         <string>6371.0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="depthLabel">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="minmagLineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="mindepthLineEdit">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="maximumLabel">
        <property name="text">
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="minimumLabel">
        <property name="text">
         <string>Minimum</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="magnitudeLabel">
        <property name="text">
         <string>Magnitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="maxmagLineEdit">
        <property name="text">
         <string>10.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="catalogTypeGroupBox">
     <property name="title">
      <string>Catalog and Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QComboBox" name="catalogComboBox">
        <item>
         <property name="text">
          <string>All Catalogs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ANF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GCMT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NEIC PDE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISC</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="typeComboBox">
        <item>
         <property name="text">
          <string>All Types</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ML</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MW</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="locationGroupBox">
     <property name="title">
      <string>Location Box</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLineEdit" name="minlonLineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="maxlonLineEdit"/>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="centerLabel">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="maxlatLineEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="minlatLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close Window</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
